CesiumGltfReader/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ target_include_directories(
     SYSTEM PUBLIC
         # Necessary for `draco/draco_features.h`
         ${CESIUM_NATIVE_DRACO_INCLUDE_DIR}
+        ${CESIUM_NATIVE_MESHOPTIMIZER_INCLUDE_DIR}
```

Comment:
Based on a quick look at meshoptimizer's CMakeLists.txt, I don't think this should be necessary. It adds it itself with `target_include_directories`.